ui/component/or-app/src/index.ts

```diff
@@ -153,6 +156,11 @@ export class OrApp<S extends AppStateKeyed> extends LitElement {
         // Set this element as the host for dialogs
         OrMwcDialog.DialogHostElement = this;
         OrMwcSnackbar.DialogHostElement = this;
+
+        // Close socket on unload/refresh of page
```

Comment:
My bad for not changing the comment after copying it from `event.ts`.

On the subject of 'whether to close the websocket connection before unload', no idea on whether it's necessary.
It's in the codebase of `event.ts` since 2019. I guess it can't hurt really right? haha